script/bootstrap.sh

```diff
@@ -138,7 +138,7 @@ else
     exit 1
 fi
 
-readonly CUR_DIR="$(dirname "$(realpath -s "$0")")"
+readonly CUR_DIR="$(dirname "$(realpath "$0")")"
```

Comment:
Is this change necessary?

Reply:
yes, for solving the following build problem
https://github.com/openthread/ot-commissioner/actions/runs/12156344538/job/33899828836